### DISCUSSION TITLE
Minio server is eating up memory like it has gone crazy (30GB)

### DISCUSSION THREAD

**Discussion by thedevd** [minio/minio#19193] [Q&A]
I have very simple usecase of using minio to store hardly object size less than 10MB, I am running minio server since a month and each day I see the memory is increase by significant amount, and currently it is consuming 30GB memory. 

I have no idea what is happening with minio  server process. 

mc version RELEASE.2023-02-28T00-12-59Z (commit-id=5fbe8c26bab5592f0bc521db00665f8670a0fb31)
Runtime: go1.19.6 linux/amd64
Copyright (c) 2015-2023 MinIO, Inc.
License GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>

**Comment by harshavardhana** [comment]
@thedevd we have no idea what you are doing, I suspect mostly you have is page-cache memory. MinIO won't generally use upto 30GiB. 

Please provide the setup and share `mc support profile alias/`

**Reply by thedevd** [reply]
@harshavardhana 

From an app, we keep sending the files in every 5 seconds towards minio cluster using mc mirror command -
```
mc alias set minio http://<minio-ip>:<minio-port> minio miniopw
mc mirror --exclude "minio*" --overwrite --remove /apps/<app-name>/data minio/<bucket-name>/apps/<app-name>
mc alias remove minio   
```

I am not able to use mc support profile command, it asks me to register license
 
```
mc alias set minio http://<minio-ip>:<minio-port> minio miniopw
mc support profile minio/
mc: <ERROR>  Please register the cluster first by running 'mc license register minio'
```

Do you know how can I configure the page-cache memory being used by minio server ?

**Reply by thedevd** [reply]
Hi @harshavardhana waiting for your reply

**Reply by harshavardhana** [reply]
Yes so please subscribe so that I can help you with your memory issues.

Thanks.
